src/components/Books.js

```diff
@@ -1,44 +1,61 @@
 import React from 'react';
 import { useDispatch, useSelector } from 'react-redux';
-import { removeBook } from '../redux/books/books';
+import { v4 as uuidv4 } from 'uuid';
+import { addBook, removeBook } from '../redux/books/books';
+import Button from './Button';
 import Form from './Form';
 
 const Books = () => {
-  const books = useSelector((state) => state.books);
+  const books = useSelector((state) => state.booksReducer.books);
   const dispatch = useDispatch();
-
-  return (
-    <div>
-      <ul>
-        {
-          books.map((book) => (
-            <li
-              key={book.id}
-            >
-              Book title:
-              {' '}
-              {book.title}
-              {' '}
-              -
-              Category:
-              {' '}
-              {book.category}
-              {' '}
-              -
-              {' '}
-              <button
-                type="button"
-                onClick={() => dispatch(removeBook(book))}
+  const submitBookToStore = () => {
+    const titleInput = document.getElementById('title');
+    const authorInput = document.getElementById('author');
+    const newBook = {
+      id: uuidv4(),
+      title: titleInput.value,
+      author: authorInput.value,
+    };
+    if (newBook.title && newBook.author) {
+      dispatch(addBook(newBook));
+      titleInput.value = '';
+      authorInput.value = '';
+    }
+  };
+  if (books.length) {
+    return (
+      <div>
+        <ul>
+          {
+            books.map((book) => (
+              <li
+                key={book.id}
               >
-                Remove book
-              </button>
-            </li>
-          ))
-        }
-      </ul>
-      <Form />
-    </div>
-
+                Book title:
+                {' '}
+                {book.title}
+                {' '}
+                -
+                Author:
+                {' '}
+                {book.author}
+                {' '}
+                -
+                {' '}
+                <Button handleSubmit={() => dispatch(removeBook(book))} buttonName="Remove book" />
+              </li>
```

Comment:
- [ ] It will be nice to create a separate component(eg Book) to help render the list of books.